### PR TITLE
Focus area updates

### PR DESCRIPTION
**Pull request by mxbclang**
Makes changes to `CODEOWNERS.md` to reflect changes to focus areas as voted on in #554 

## Checklist

- [x] PR has either `[Focus]` or `Infrastructure` label.
- [x] PR has a `[Type]` label.
- [x] PR has a milestone or the `no milestone` label.

### CODE REVIEW

**Review by felixarntz — COMMENTED** [review]
@bethanylang We should probably remove the Site Health focus from this file too and rename the JavaScript focus accordingly. For the latter, we still have to decide what name to go with - maybe "CSS and JS" (`css-and-js`) is the most clear?

Or was your idea that we would make those changes together with the PR that migrates the module directories etc? (see also my comment https://github.com/WordPress/performance/issues/554#issuecomment-1282809582)

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -35,22 +39,22 @@
 /tests/modules/images/webp-uploads                                          @adamsilverstein @felixarntz @mitogh
 /tests/testdata/modules/images/webp-uploads                                 @adamsilverstein @felixarntz @mitogh
 
-# Module: WebP Support Health Check
+# Module: WebP Support
```
Comment (felixarntz) [inline .github/CODEOWNERS]: The original name here was correct, as that is also the name of the module.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -35,22 +39,22 @@
...
 /modules/site-health/webp-support                                           @adamsilverstein @kirtangajjar @mitogh
 /tests/modules/site-health/webp-support                                     @adamsilverstein @kirtangajjar @mitogh
 /tests/testdata/modules/site-health/webp-support                            @adamsilverstein @kirtangajjar @mitogh
 
-# Module: Autoloaded Options Health Check
+# Module: Audit Autoloaded Options
```
Comment (felixarntz) [inline .github/CODEOWNERS]: See above.

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -35,22 +39,22 @@
...
 /modules/site-health/audit-autoloaded-options                               @manuelRod
 /tests/modules/site-health/audit-autoloaded-options                         @manuelRod
 /tests/testdata/modules/site-health/audit-autoloaded-options                @manuelRod
 
-# Module: Enqueued Assets Health Check
+# Module: Audit Enqueued Assets
```
Comment (felixarntz) [inline .github/CODEOWNERS]: See above.

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -35,22 +39,22 @@
...
 /modules/site-health/audit-enqueued-assets                                  @manuelRod
 /tests/modules/site-health/audit-enqueued-assets                            @manuelRod
 /tests/testdata/modules/site-health/audit-enqueued-assets                   @manuelRod
 
-# Module: Persistent Object Cache Health Check
+# Module: Persistent Object Cache
```
Comment (felixarntz) [inline .github/CODEOWNERS]: See above.

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -60,6 +64,6 @@
 /tests/modules/images/dominant-color                                        @pbearne @spacedmonkey
 /tests/testdata/modules/images/dominant-color                               @pbearne @spacedmonkey
 
-# Module: Full Page Cache Health Check
+# Module: Audit Full Page Cache
```
Comment (felixarntz) [inline .github/CODEOWNERS]: See above.

**Review by felixarntz — COMMENTED** [review]
@bethanylang Could you please revert the name changes of the Site Health modules in here? The current names that those have are the correct module names.

**Comment by mxbclang** [comment]
@felixarntz I've reverted that commit; sorry for the confusion!

File: .github/CODEOWNERS [hunk 6]
```diff
@@ -1,16 +1,20 @@
...
+/tests/modules/database
+/tests/testdata/modules/database
+
 # Focus: Images
 /modules/images                             @adamsilverstein @getsource
 /tests/modules/images                       @adamsilverstein @getsource
 /tests/testdata/modules/images              @adamsilverstein @getsource
 
-# Focus: JavaScript
+# Focus: CSS & JS
```
Comment (felixarntz) [inline .github/CODEOWNERS]: I think here is where I saw the "CSS & JS", but you mentioned it should be "JS & CSS" :)

**Review by felixarntz — COMMENTED** [review]
@bethanylang One more thing here.

File: .github/CODEOWNERS (hunk 6 above)
Comment (mxbclang) [reply .github/CODEOWNERS]: @felixarntz Fixed!

File: .github/CODEOWNERS [hunk 7]
```diff
@@ -1,16 +1,20 @@
...
-/docs                                       @JustinyAhin @felixarntz
 /admin                                      @JustinyAhin @felixarntz
 /tests/admin                                @JustinyAhin @felixarntz
 /bin                                        @JustinyAhin @felixarntz
 /.github                                    @JustinyAhin @felixarntz
 
+# Focus: Database
+/modules/database
+/tests/modules/database
+/tests/testdata/modules/database
```
Comment (felixarntz) [inline .github/CODEOWNERS]: Should @OllieJones be added here now? :)

**Review by felixarntz — APPROVED** [review]
LGTM, just one question.

File: .github/CODEOWNERS (hunk 7 above)
Comment (mxbclang) [reply .github/CODEOWNERS]: Sorry, missed your Slack mention. Updated!

**Review by mukeshpanchal27 — COMMENTED** [review]
LGTM. The PR shows errors for the CODEOWNERS.

Unknown owner on line 8/9/10: make sure @olliejones exists and has write access to the repository.

**Comment by mxbclang** [comment]
Thanks for flagging @mukeshpanchal27, let me check with the meta team about that!

File: phpcs.xml.dist [hunk 8]
```diff
@@ -18,10 +18,13 @@
 	<file>./modules</file>
 	<file>./tests</file>
 
-	<!-- Do not require docblocks for unit tests -->
+	<!-- Do not require file headers on generated files -->
 	<rule ref="Squiz.Commenting.FileComment.WrongStyle">
+		<exclude-pattern>default-enabled-modules.php</exclude-pattern>
 		<exclude-pattern>module-i18n.php</exclude-pattern>
 	</rule>
```
Comment (felixarntz) [inline phpcs.xml.dist:25]: This change is unrelated to this PR, but I got pre-commit errors about this, and this file should also have that rule excluded.

**Comment by felixarntz** [comment]
@mukeshpanchal27 Could you please give this another look and approve/merge if things look good? @OllieJones is now a member of the org, so the `CODEOWNERS` is now valid. I've also made super minor tweaks here to the indentation and a PHPCS fix.

**Review by mukeshpanchal27 — APPROVED** [review]
Thanks @bethanylang LGTM.

**Pull request merged**